src/api/app.ts

```diff
@@ -111,7 +112,7 @@ export function create(
             stream.on('close', () => observer.complete());
             stream = stream.pipe(new JSONStream());
             stream.on('data', (data) => data && observer.next(data));
-        });
+        }).flatMap(deCompressNodes);
```

Comment:
The `updatesSource` returns Observable of `{ object: models.Workflow, eventType: string  } `, so deCompressNodes should be executed on `object` field.

I know that `flatMap` works with function which returns Observable, not Promise. Does it work with Promise too?

If you change apply both changes that code might looks like this:

```
}).flatMap((change) => Observable.fromPromise(deCompressNodes(change.object).then((workflow) => ({...change, object: workflow}))));
```